### ISSUE TITLE
swag-dashboard: how to access this mod?

### ISSUE THREAD

**Issue by ranomier** [linuxserver/docker-mods#303]
So yeah i can go into the container and run the goaccess binary myself,
but how to expose it to the internet?

I'm using authentik, i would put login in front of it.

**Comment by nishantbb** [comment]
I am also curious and feeling dumb. Does this mod provide a dashboard for each proxied site? And not a single dashboard for the various proxied sites?

Should I be trying to access the dashboard over the internet for site1.com at dashboard.site1.com ? If so, is there a way to deploy this as for the local network only? Apologies for my confusion

**Comment by aptalca** [comment]
It's a single dashboard for your server. By default it is published at `https://dashboard.yourdomain.com` and is only accessible from your lan. But you can change all that by editing the `dashboard.subdomain.conf` in your proxy-confs folder.

**Comment by nishantbb** [comment]
Thanks, I didn't realize the proxy conf was added, and now I see the IP restrictions for the local network. I tried updating the server name and adding the appropriate subdomains to my Swag and DuckDNS containers, but when I access the domain it returns a 403. I am trying from a computer on the same network as the server, and I've tried whitelisting this machine's local IP address as well.

Is there a way to host the dashboard on a particular port, like a typical docker container? Or something I could use PiHole DNS records to help? For example, I reach most of my services at http://servername:PORT

I am trying to reach the dashboard at domainfordashboard.duckdns.org

`      - URL=duckdns.org
      - SUBDOMAINS=otherstuff,domainfordashboard
      - VALIDATION=http
      - ONLY_SUBDOMAINS=true
      - DOCKER_MODS=linuxserver/mods:swag-maxmind|linuxserver/mods:swag-dashboard`

changed portion of proxy conf
`server_name domainfordashboard.duckdns.org;`

Thank you.
